keep/api/models/db/migrations/versions/2024-09-12-14-26_9af44b2de182.py

```diff
@@ -0,0 +1,21 @@
+"""empty message
```

Comment:
is there a reason why there is a merge migration instead of deleting the 09-08 migration and recreate it from head?

Reply:
Removed